recipes/rocrate/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "rocrate" %}
+{% set version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/rocrate-{{ version }}.tar.gz
+  sha256: 4e67606965ed7a45c845222bbd60bdd766886e17f21d8b603e0ae68c8cc71f2a
+
+build:
+  entry_points:
+    - rocrate=rocrate.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - requests
+    - arcp ==0.2.1
```

Could you use the following as the 0.2.1 is not yet available ?
```suggestion
    - arcp ==0.2.0
```